app/main.py

```diff
@@ -1,8 +1,28 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.wife = None
+        self.husband = None
+        self.add_instance()
+
+    def add_instance(self) -> None:
+        Person.people[self.name] = self
```

Comment:
```suggestion
        self[name] = self
```

Reply:
Thank You!

Reply:
Shame to me, but so far I can't understand  meaning self[name] = self

Reply:
please, if you do not understand, please sign up at Q&A, or ask a question in the Slack chat